third_party/boost/LICENSE.md

```diff
@@ -0,0 +1,23 @@
+Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
Is Boost license compatible with the rest of licenses used by Open Telemetry?

Reply:
boost 1.0 is a small, permissible license. I don't see it being problematic.

Reply:
ok, we were in general avoiding boost elsewhere due to size implications..
I do not have a very strong opinion on this. I would've avoided it if we can.

Reply:
It's using the boost license, not the boost libraries.